.rubocop.yml

```diff
@@ -2,6 +2,7 @@ inherit_from: .rubocop_todo.yml
 
```

Comment:
Hey @kgorazd I had some time on weekend and decided to improve a bit a quality of code (of tests mostly), it's first commit only, I'll push more, may be on the next weekend. So, if you have some time, it'll be great if you check this commit.
Thanks

Reply:
Sure. All good so far 👍